androidframer/framer.py

```diff
@@ -62,33 +76,51 @@ def start(self):
 							tempOut = self.label(framed, title1, title2)
 							out = tempOut.replace(imgFolder, outFolder)
 							os.rename(tempOut, out)
-
-
+		logging.info("Finished running in {}s".format(round(time.time()-start_time,2)))
 
 	def cmd(self, command):
+		'''Opens pipe and reads command from system'''
 		return os.popen(command).read()
 
 	def resize(self, imgFolder, file, ext):
+		"""
+		Resize image.
+
+		Takes file in imgFolder and resizes it.
+
+		Parameters
+		----------
+		imgFolder : string
+			Address of folder with image
+		file : string
+			Name of image file
+		ext : string
+			Extension of image file
+
+		Returns
+		-------
+		string
+			file with an appended underscore
+		"""
 		img = os.path.join(imgFolder, (file + ext))
 		out = os.path.join(imgFolder, (file + '_' + ext))
 		command = f"convert {img} -resize %{str(self.resizeRatio)} {out}"
 		self.cmd(command)
 		return file + "_"
 
 	def frame(self, imgFolder, file, ext):
+		'''Put the file.ext in a frame'''
 		img = os.path.join(imgFolder, (file + ext))
 		out = os.path.join(imgFolder, (file + 'framed' + ext))
 		command = f"convert {self.bg} {img} -geometry +{self.xPos}+{self.yPos} -composite {out}"
 		self.cmd(command)
 		return out
 
 	def label(self, img, title1, title2):
+		'''Label img with title1 and title2'''
 		out = img.replace('_framed', '_out')
 		command = f"convert {img} -font {self.font} -gravity North -fill white -pointsize {self.fontSize} -draw \"text 0,100 '{title1}'\" -draw \"text 0,220 '{title2}'\" {out}"
 		self.cmd(command)
 		os.remove(img)
 		os.remove(img.replace('_framed', '_'))
-		return out
-
-if __name__== '__main__':
-	Framer().start()
```

Comment:
Framer().start method is removed. It should be called somewhere.

Reply:
Since Framer is being published on PyPi, I thought the end-user ought to initialise a Framer instance themselves and call the start method rather than providing this in the package itself?

Reply:
All right. I published and tried with a sample. I forgot to delete this comment :)